### PR TITLE
Fix typo

### PR DESCRIPTION
**Pull request by Neizvestnyj**
All changes have been checked.

### CODE REVIEW

File: kivymd/uix/dialog.py [hunk 1]
```diff
@@ -155,12 +155,12 @@ def show_alert_dialog(self):
                 adaptive_height: True
                 cols: 1
 
-        BoxLayout:
+        MDBoxLayout:
             id: spacer_bottom_box
             size_hint_y: None
             height: self.minimum_height
```
Comment (HeaTTheatR) [inline kivymd/uix/dialog.py:162]: adaptive_height: True

File: kivymd/uix/imagelist.py [hunk 2]
```diff
@@ -148,7 +148,7 @@ def build(self):
         x: root.x
         y: root.y if root.overlap or root.box_position == 'header' else box.top
 
-    BoxLayout:
+    MDBoxLayout:
```
Comment (HeaTTheatR) [inline kivymd/uix/imagelist.py:151]: Using MDBoxLayout instead of BoxLayout needs to be tested.

File: kivymd/uix/imagelist.py [hunk 3]
```diff
@@ -175,7 +175,7 @@ def build(self):
...
```
Comment (HeaTTheatR) [inline kivymd/uix/imagelist.py:178]: Using MDBoxLayout instead of BoxLayout needs to be tested.

File: kivymd/uix/list.py [hunk 4]
```diff
@@ -550,7 +550,7 @@ def build(self):
             pos: self.pos
             size: self.size
 
-    BoxLayout:
+    MDBoxLayout:
```
Comment (HeaTTheatR) [inline kivymd/uix/list.py:553]: Using MDBoxLayout instead of BoxLayout needs to be tested.

File: kivymd/uix/list.py [hunk 5]
```diff
@@ -597,7 +597,7 @@ def build(self):
 
 <OneLineAvatarListItem>
 
-    BoxLayout:
+    MDBoxLayout:
```
Comment (HeaTTheatR) [inline kivymd/uix/list.py:600]: Using MDBoxLayout instead of BoxLayout needs to be tested.

File: kivymd/uix/list.py [hunk 6]
```diff
@@ -607,7 +607,7 @@ def build(self):
 
 <ThreeLineAvatarListItem>
 
-    BoxLayout:
+    MDBoxLayout:
```
Comment (HeaTTheatR) [inline kivymd/uix/list.py:610]: Using MDBoxLayout instead of BoxLayout needs to be tested.

File: kivymd/uix/list.py [hunk 7]
```diff
@@ -617,7 +617,7 @@ def build(self):
 
 <OneLineIconListItem>
 
-    BoxLayout:
+    MDBoxLayout:
```
Comment (HeaTTheatR) [inline kivymd/uix/list.py:620]: Using MDBoxLayout instead of BoxLayout needs to be tested.

File: kivymd/uix/list.py [hunk 8]
```diff
@@ -626,7 +626,7 @@ def build(self):
 
 
 <ThreeLineIconListItem>
-    BoxLayout:
+    MDBoxLayout:
```
Comment (HeaTTheatR) [inline kivymd/uix/list.py:629]: Using MDBoxLayout instead of BoxLayout needs to be tested.

File: kivymd/uix/list.py [hunk 9]
```diff
@@ -636,7 +636,7 @@ def build(self):
 
 <OneLineRightIconListItem>
 
-    BoxLayout:
+    MDBoxLayout:
```
Comment (HeaTTheatR) [inline kivymd/uix/list.py:639]: Using MDBoxLayout instead of BoxLayout needs to be tested.

File: kivymd/uix/list.py [hunk 10]
```diff
@@ -646,7 +646,7 @@ def build(self):
 
 <ThreeLineRightIconListItem>
 
-    BoxLayout:
+    MDBoxLayout:
```
Comment (HeaTTheatR) [inline kivymd/uix/list.py:649]: Using MDBoxLayout instead of BoxLayout needs to be tested.

File: kivymd/uix/list.py [hunk 11]
```diff
@@ -656,7 +656,7 @@ def build(self):
 
 <OneLineAvatarIconListItem>
 
-    BoxLayout:
+    MDBoxLayout:
```
Comment (HeaTTheatR) [inline kivymd/uix/list.py:659]: Using MDBoxLayout instead of BoxLayout needs to be tested.

File: kivymd/uix/list.py [hunk 12]
```diff
@@ -666,7 +666,7 @@ def build(self):
 
 <TwoLineAvatarIconListItem>
 
-    BoxLayout:
+    MDBoxLayout:
```
Comment (HeaTTheatR) [inline kivymd/uix/list.py:669]: Using MDBoxLayout instead of BoxLayout needs to be tested.

File: kivymd/uix/list.py [hunk 13]
```diff
@@ -676,7 +676,7 @@ def build(self):
 
 <ThreeLineAvatarIconListItem>
 
-    BoxLayout:
+    MDBoxLayout:
```
Comment (HeaTTheatR) [inline kivymd/uix/list.py:679]: Using MDBoxLayout instead of BoxLayout needs to be tested.

File: kivymd/uix/picker.py [hunk 14]
```diff
@@ -540,7 +540,7 @@ def show_theme_picker(self):
 
         # TODO: Replace the GridLayout with a RecycleView
         # if it improves performance.
-        GridLayout:
+        MDGridLayout:
```
Comment (HeaTTheatR) [inline kivymd/uix/picker.py:543]: Using MDGridLayout instead of GridLayout needs to be tested.

File: kivymd/uix/imagelist.py (hunk 2 above)
Comment (Neizvestnyj) [reply kivymd/uix/imagelist.py:151]: I tested this using all the examples from the documentation

**Comment by HeaTTheatR** [comment]
@Neizvestnyj The "Added typing to class method" is not the same as the changes you made.

**Comment by Neizvestnyj** [comment]
> @Neizvestnyj The "Added typing to class method" is not the same as the changes you made.

I thought to name it the same way as the branch, but I think this is also wrong

File: kivymd/uix/dialog.py (hunk 1 above)
Comment (Neizvestnyj) [reply kivymd/uix/dialog.py:162]: Yes, exactly, it's the same thing)

Comment (Neizvestnyj) [reply kivymd/uix/dialog.py:162]: Yes, exactly, it's the same thing)

**Comment by Neizvestnyj** [comment]
I can create a new branch and additionally test it on android, although I think there will be no difference

**Comment by HeaTTheatR** [comment]
@Neizvestnyj Yes, tests are needed. And the "Added typing to class method" commits don't match your changes

**Comment by Neizvestnyj** [comment]
I remembered why I didn't set `adaptive_height: True`, just then the dialog is displayed incorrectly, it becomes higher in the standard mode...

**Pull request closed without merging**